SU2_CFD/src/solvers/CMeshSolver.cpp

```diff
@@ -661,13 +662,15 @@ void CMeshSolver::SetBoundaryDisplacements(CGeometry *geometry, CNumerics *numer
   /*--- Symmetry plane is clamped, for now. ---*/
   for (iMarker = 0; iMarker < config->GetnMarker_All(); iMarker++) {
     if ((config->GetMarker_All_Deform_Mesh(iMarker) == NO) &&
+        (config->GetMarker_All_Deform_Mesh_Sym_Plane(iMarker) == NO) &&
         (config->GetMarker_All_Moving(iMarker) == NO) &&
         (config->GetMarker_All_KindBC(iMarker) == SYMMETRY_PLANE)) {
 
       BC_Clamped(geometry, numerics, config, iMarker);
     }
```

Comment:
ahaha yes I also found it quite funny when I first saw it. But it is fine now to clamp it... As the sym plane for fluid and mesh are now separated, one will explicity use the deform mesh sym plane marker if she/he wants to obtain a symmetric deformation. Otherwise, the code will clamp deformation.

Reply:
That was because the FEA solver did not use to have a symmetry BC.
And then when I added it, it was better to clamp the symmetry plane for my testcases :)
But now that Nicola decoupled the physical symmetry from the deformation symmetry all is well xD